django/views/i18n.py

```diff
@@ -29,8 +29,9 @@ def builtin_template_path(name):
 
 def set_language(request):
     """
-    Redirect to a given URL while setting the chosen language in the session
```

Comment:
Elsewhere (other than the other place in this PR), this is just "the language cookie", so I think this is sufficient here:
```suggestion
    Redirect to a given URL while setting the chosen language in the language cookie.
```